src/js/components/Diagram/index.d.ts

```diff
@@ -17,10 +17,16 @@ export type DiagramAnimationType =
 
 export interface DiagramProps {
   animation?: DiagramAnimationType;
+  arrow?: {
+    fill?: boolean;
+    color?: ColorType;
+  };
   connections: {
     anchor?: DiagramConnectionAnchor;
     animation?: DiagramAnimationType;
     color?: ColorType;
+    decreaseLineLength?: Number;
```

Comment:
I don't think the caller should have to figure this out. Instead, I think as part of measuring where the `fromTarget` and `toTarget` are on screen, in combination with `anchor` indicating how to draw the lines, we can calculate the appropriate edge location. For instance, if the center of the fromTarget is 100,100, the anchor is horizontal, and the size of the fromTarget is 20x20, we would draw the line starting at 110,100.

Reply:
![Screenshot 2022-03-21 203413](https://user-images.githubusercontent.com/61521805/159385311-dca191cf-9bab-4a76-aaf6-2846a3ccb70e.png)

If I don't use the `decreaseLineLength`, then shows over the target element. The `decreaseLineLength` is being used alongside the `elementPosition` prop which represents the targetElement (i.e, if it is at top or bottom) and based on this prop we provide the -ve or +ve value to decrease the line length. Please, check line number 46-48 for reference.

```
    const lineLength = ['top'].includes(elementPosition)
      ? decreaseLineLength
      : -decreaseLineLength;
```

However, I didn't quite understand how to do by your mentioned way by modifying into fromTarget values. Even if we did that, I imagine it'll be a breaking change ?
Please, let me know if I am missing something.

Reply:
Regarding `elementPosition` If I have toTarget A centered at location 100,100 and fromTarget B centered at 200,200, each with height and width 20, I'm not sure what setting `elementPosition: 'top'` indicates. Does that mean the line should terminate at the top of B and the top of A?

My suggestion is to have a way to indicate which side of each target to orient the line from. In my example above, I think we need a way to describe whether to terminate the line at 90,100 for A or 100,110 for A. Same for B. Perhaps something like:
```
connections: [
  {
    from: { target: A, anchor: 'horizontal' },
    to: { target: B, anchor: 'vertical', arrow: true },
    type: 'rectilinear',
  }
],

```
which, given the coordinates and dimensions above, would draw from 90,100 to 200,100 to 200, 190.

Reply:
@ericsoderberghp I have added your specified approach, but seems like it doesn't to work with the `horizontal` anchor when we have to select it from `COMMANDS`. Please, take a look and let me know if I am missing something.